### PR TITLE
PDF Renderer: Add border to content, move controls to top

### PR DESCRIPTION
**Pull request by nucleogenesis**
### Summary

**RE: https://github.com/learningequality/kolibri/issues/6901**

- Full screen toggle, zoom in and out are all move to a top bar on PDFs.
- Bar hides when you scroll down, shows when you scroll up.

hide-show-on-scroll

**RE: https://github.com/learningequality/kolibri/issues/6909**

- Adds 2dp drop shadow to container of renderer.
- Honestly I have no idea exactly what is best here but I felt like a solid border kind of looked odd. But some shadow seemed nice and to have the expected effect.

_Solid Black 1px_

border-1px-mobile

_2dp Drop Shadow_

shadow-2dp-mobile

Related to the above:

- CSS to ensure hiding & showing of top control bar doesn't effect a covering of information beneath the PDF (ie, author's name, download button).

### Reviewer guidance

Test this:

- Full screen & not fullscreen & mobile & not mobile & zoom.

What do you think about this:

- Solid border vs shadow? 
- Does the hiding controls violate any kind of a11y concerns?

### References

https://github.com/learningequality/kolibri/issues/6909
https://github.com/learningequality/kolibri/issues/6901

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by bjester** [comment]
I'm unable to get any PDF's to load using the pex. I've tried various different channels and PDF's. It sits there spinning/loading, and eventually says `Kolibri is unable to render this resource`. No errors in the console.

**Comment by rtibbles** [comment]
This would be caused by an error during the dynamic loading of the PDF renderer - odd that there are no errors in the console though.

**Comment by bjester** [comment]
@rtibbles I think it could be that something is wrong with the build. I noticed that this PR should also contain fixes from https://github.com/learningequality/kolibri/pull/7006 but that issue is still broken. I also tried the pex from that PR and seems that issue is still broken for me.

**Comment by rtibbles** [comment]
I am guessing you didn't mean #7008 as that is this PR?

**Comment by bjester** [comment]
> I am guessing you didn't mean #7008 as that is this PR?

Whoops, I meant https://github.com/learningequality/kolibri/pull/7006

File: kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue [hunk 1]
```diff
@@ -378,4 +400,31 @@
...
+      top: 8px;
+    }
+  }
+
+  .fullscreen-header {
+    height: $controls-height;
+    text-align: end;
+  }
+
+  .slide-enter-active {
```
Comment (jonboiser) [inline kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue:423]: Let's use the same material design easing functions as the scrolling header e.g.

https://github.com/learningequality/kolibri/blob/0498cfd39a6c1d70e5646470a91b515619d03cd2/kolibri/core/assets/src/views/CoreBase/ScrollingHeader.vue#L150-L160

Comment (jonboiser) [reply kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue:423]: These need to be imported from the design system `@import '~kolibri-design-system/lib/styles/definitions';`

**Review by bjester — CHANGES_REQUESTED** [review]
I'm unable to build this from your branch, which now I think explains why PDF's simply don't load for me:
```
Kolibri Build: Failed to compilekolibri.plugins.pdf_viewer.main bundle!
./kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue?vue&type=style&index=0&id=c264e7fe&lang=scss&scoped=true&)
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/sass-loader/lib/loader.js):

      },
^
      File to import not found or unreadable: ~kolibri-design-system/lib/styles/definitions.
      in /home/bjester/Projects/learningequality/kolibri/kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue (line 354, column 1)

Kolibri Production Build: There was a build error for kolibri.plugins.pdf_viewer.main
Kolibri Production Build: 
ERROR in ./kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue?vue&type=style&index=0&id=c264e7fe&lang=scss&scoped=true& (./node_modules/mini-css-extract-plugin/dist/loader.js!./node_modules/css-loader??ref--4-1!./node_modules/vue-loader/lib/loaders/stylePostLoader.js!./node_modules/postcss-loader/lib??ref--4-2!./node_modules/sass-loader/lib/loader.js??ref--4-3!./node_modules/vue-loader/lib??vue-loader-options!./node_modules/svg-icon-inline-loader!./kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue?vue&type=style&index=0&id=c264e7fe&lang=scss&scoped=true&)
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/sass-loader/lib/loader.js):

      },
^
      File to import not found or unreadable: ~kolibri-design-system/lib/styles/definitions.
      in /home/bjester/Projects/learningequality/kolibri/kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue (line 354, column 1)
    at runLoaders (/home/bjester/Projects/learningequality/kolibri/node_modules/webpack/lib/NormalModule.js:313:20)
    at /home/bjester/Projects/learningequality/kolibri/node_modules/loader-runner/lib/LoaderRunner.js:367:11
    at /home/bjester/Projects/learningequality/kolibri/node_modules/loader-runner/lib/LoaderRunner.js:233:18
    at context.callback (/home/bjester/Projects/learningequality/kolibri/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at Object.render [as callback] (/home/bjester/Projects/learningequality/kolibri/node_modules/sass-loader/lib/loader.js:52:13)
    at Object.done [as callback] (/home/bjester/Projects/learningequality/kolibri/node_modules/neo-async/async.js:8067:18)
    at options.error (/home/bjester/Projects/learningequality/kolibri/node_modules/node-sass/lib/index.js:294:32)
 @ ./kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue?vue&type=style&index=0&id=c264e7fe&lang=scss&scoped=true& 1:0-579 1:595-598 1:600-1176 1:600-1176
 @ ./kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue
 @ ./kolibri/plugins/pdf_viewer/assets/src/module.js
```

**Comment by nucleogenesis** [comment]
@bjester I pushed a fix for your issue. Apparently I never `yarn unlink` 'ed the KDS package while switching branches so it was found in my node_modules :-/

**Comment by bjester** [comment]
@nucleogenesis Since the control bar hides when you scroll, I think it would make sense for it to overlay the PDF view if possible. Right now, when I scroll down, the bar then hides, but the PDF content then moves within the viewport because of the bar disappearing. This is slightly jarring in my opinion. An alternate solution would be to scroll the viewport the same amount so the PDF content stays in the same spot within the viewport.

Another note is whether that behavior is needed when not in full screen mode?

**Comment by nucleogenesis** [comment]
Thanks @bjester that's a great point. I've made it `position:absolute` which made for a much cleaner transition animation as well.

Just pushed the change if you could give it a look again?

As for whether it's necessary in full screen I'll defer to others ( cc @indirectlylit @jtamiace )

**Review by bjester — APPROVED** [review]
Good steps forward!

**Comment by indirectlylit** [comment]
awesome. I think the drop-shadow works well

**Pull request merged**